### DISCUSSION TITLE
WalletConnect meet MATRIX ?

### DISCUSSION THREAD

**Discussion by buschmann** [WalletConnect/walletconnect-monorepo#5294] [Ideas]
Good morning++

Can anyone tell me how complex that will be ??
Is there any group for ?
![image](https://github.com/user-attachments/assets/2faaef98-f981-4b67-8e8a-d36a28675f40)

Regards Buschi Buschmann.

**Comment by yncyrydybyl** [comment]
There is a project which has written a keycloak module for some Walletintegration. https://github.com/ineat/keyblock
